graphql/config_resolver.go

```diff
@@ -0,0 +1,21 @@
+package graphql
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.36
+
+import (
+	"context"
+
+	"github.com/evergreen-ci/evergreen/rest/model"
+)
+
+// Port is the resolver for the port field.
+func (r *containerPoolResolver) Port(ctx context.Context, obj *model.APIContainerPool) (int, error) {
+	return int(obj.Port), nil
+}
```

Comment:
This file is not generated. I wasn't really sure how to handle `uint16` through GraphQL, but I casted it to an `int` because I figured that given the max possible value of `uint16`, it should be able to fit within the `int` type...

Reply:
`uint16` isn't significant in this case, I believe it was just originally used to prevent inputting negative numbers. The conversion LGTM!